set.mm

```diff
@@ -30855,6 +30902,20 @@ Such interpretation is rarely needed (see also ~ df-ral ).  (Contributed
       BDEJZKACELBDELRTSUAABCDEFGHMABCDEFGHNOACEPBDEPQ $.
   $}
 
+  ${
+    $d x y A $.  $d y ph $.  $d x ps $.
+    cbvralvw.1 $e |- ( x = y -> ( ph <-> ps ) ) $.
+    $( Version of ~ cbvralv with a disjoint variable condition, which does not
+       require ~ ax-13 .  (Contributed by Gino Giotto, 10-Jan-2024.) $)
+    cbvralvw $p |- ( A. x e. A ph <-> A. y e. A ps ) $=
+      ( nfv cbvralw ) ABCDEADGBCGFH $.
+
+    $( Version of ~ cbvrexv with a disjoint variable condition, which does not
+       require ~ ax-13 .  (Contributed by Gino Giotto, 10-Jan-2024.) $)
+    cbvrexvw $p |- ( E. x e. A ph <-> E. y e. A ps ) $=
+      ( nfv cbvrexw ) ABCDEADGBCGFH $.
```

Comment:
for these two even more axioms can be saved by using cbvalvw and cbvexvw

Reply:
Done (took me more time than I would like to admit to find the proofs).